### PR TITLE
Changes to Free tier

### PR DESCRIPTION
**Pull request by LalithManokar**

### CODE REVIEW

File: tutorials/btp-hyperscaler-extension/btp-hyperscaler-extension.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ primary_tag: products>sap-btp--cloud-foundry-environment
 ---
 
 ## Prerequisites
- - You have either an [enterprise](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/171511cc425c4e079d0684936486eee6.html) or a [trial](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/046f127f2a614438b616ccfc575fdb16.html) global account in SAP BTP
+- Get an Account on SAP BTP to Try Out Free Tier Service Plans.follow steps 1 to 8 from this tutorial [Get an Account on SAP BTP](btp-free-tier-account)
```
Comment (MichaelCzcz) [inline tutorials/btp-hyperscaler-extension/btp-hyperscaler-extension.md:11]: A prerequisite is not an instruction! Please rephrase!

File: tutorials/btp-hyperscaler-extension/btp-hyperscaler-extension.md [hunk 2]
```diff
@@ -8,11 +8,11 @@ primary_tag: products>sap-btp--cloud-foundry-environment
...
  - You have an [account on the SAP Gateway Demo System (ES5)](gateway-demo-signup) created
  - You have [connected SAP BTP to your SAP Gateway Demo System (ES5) account](cp-portal-cloud-foundry-gateway-connection)
  - [Install Docker](https://docs.docker.com/get-docker/) (or any other Container tool).
- - Create an account at (https://hub.docker.com/) (or any other Container Image Library).
+ - Create an account at [hub](https://hub.docker.com/) (or any other Container Image Library).
```
Comment (MichaelCzcz) [inline tutorials/btp-hyperscaler-extension/btp-hyperscaler-extension.md:15]: A prerequisite is not an instruction! Please rephrase!

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Please update the tag namespaces (products -> software-product)!

File: tutorials/btp-hyperscaler-extension/btp-hyperscaler-extension.md (hunk 2 above)
Comment (LalithManokar) [reply tutorials/btp-hyperscaler-extension/btp-hyperscaler-extension.md]: I Have made the changes please have a look

File: tutorials/btp-hyperscaler-extension/btp-hyperscaler-extension.md (hunk 1 above)
Comment (LalithManokar) [reply tutorials/btp-hyperscaler-extension/btp-hyperscaler-extension.md]: I have made the changes please have a look

**Pull request merged**